IPython/lib/display.py

```diff
@@ -342,7 +343,7 @@ def __init__(self,
     def _format_path(self):
         fp = ''.join([self.url_prefix,self.path])
```

Comment:
It probably needs to be escaped on this line too - this is what goes into the `href`.